zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/GroupCreatedUpdatedHandlerTest.java

```diff
@@ -56,7 +56,7 @@ void shouldHandleRecord() {
   void shouldGenerateIds() {
     // given
     final Record<GroupRecordValue> groupRecord =
-        factory.generateRecordWithIntent(ValueType.GROUP, GroupIntent.DELETED);
+        factory.generateRecordWithIntent(ValueType.GROUP, GroupIntent.CREATED);
```

Comment:
🔧 You should add another similar test case to verify the UPDATED intent, or you can create a parameterized test